src/battle_util.c

```diff
@@ -4158,23 +4157,14 @@ bool8 ChangeTypeBasedOnTerrain(u8 battlerId)
     return TRUE;
 }
 
-// Supreme Overlord adds a damage boost for each fainted ally.
-// The first ally adds a x1.2 boost, and subsequent allies add an extra x0.1 boost each.
+// Supreme Overlord adds a x0.1 damage boost for each fainted ally.
 static u16 GetSupremeOverlordModifier(u8 battlerId)
 {
     u32 i;
-    struct Pokemon *party = GetBattlerParty(battlerId);
     u16 modifier = UQ_4_12(1.0);
-    bool8 appliedFirstBoost = FALSE;
 
-    for (i = 0; i < PARTY_SIZE; i++)
-    {
-        if (GetMonData(&party[i], MON_DATA_SPECIES) != SPECIES_NONE
-         && !GetMonData(&party[i], MON_DATA_IS_EGG)
-         && GetMonData(&party[i], MON_DATA_HP) == 0)
-            modifier += (!appliedFirstBoost) ? UQ_4_12(0.2) : UQ_4_12(0.1);
-        appliedFirstBoost = TRUE;
-    }
+    for (i = 0; i < gBattleStruct->supremeOverlordCounter[battlerId]; i++)
+        modifier += UQ_4_12(0.1);
```

Comment:
Won't this cap at x1.5? Supreme Overlord caps at x1.6, and as explained in the comment that you removed, the first ally that faints provides a x0.2 boost, while each other member of the party add a x0.1 to the multiplier when they faint.

Reply:
> Won't this cap at x1.5?

[Yes](https://www.smogon.com/forums/threads/scarlet-violet-battle-mechanics-research.3709545/post-9421520).